backend/src/org/commcare/xml/SuiteParser.java

```diff
@@ -118,11 +117,6 @@ public Suite parse() throws InvalidStructureException, IOException, XmlPullParse
             // TODO Auto-generated catch block
             e.printStackTrace();
             throw new InvalidStructureException("Pull Parse Exception, malformed XML.", parser);
-        } catch (StorageFullException e) {
-            e.printStackTrace();
-            //BUT not really! This should maybe be added to the high level declaration
-            //instead? Or maybe there should be a more general Resource Management Exception?
-            throw new InvalidStructureException("Problem storing parser suite XML", parser);
```

Comment:
It seems like the StorageFullException was being used here to indicate some other actual problem. We may not want to lose that?